### PR TITLE
Instructions on how to run the project

### PR DESCRIPTION
**Pull request by danielepolencic**
This PR includes instructions on how to set up a local environment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+Attach to the Localstack container with:
+
+```bash
+docker exec -ti <a51bc3adcd34> bash
+```
+
+Load the data with:
+
+```bash
+/docker-entrypoint-initaws.d/setup.sh
```
Comment (matt-crowson-moj) [inline README.md:41]: `docker-compose up` automatically does this for you so no need to connect and load manually.  Well it should do this.  Does it not for you?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+```
+
+Once the command completes, you can close the session.
+
+Head over to Intellij and configure the profile to use Localstack in `Run > Edit configurations`:
```
Comment (matt-crowson-moj) [inline README.md:46]: can run from command line with `./gradlew bootRun --args='--spring.profiles.active=localstack'`.  Might not be using Intellij (though lets be honest if you are writing in Kotlin then that is highly unlikely).

**Review by matt-crowson-moj — APPROVED** [review]
A good start.  I should have done this myself really.  It was on my list of TODO's

File: README.md (hunk 1 above)
Comment (danielepolencic) [reply README.md:41]: I couldn't see where the `setup.sh` file is called. Is it executed by the Localstack Docker container?

File: README.md (hunk 2 above)
Comment (danielepolencic) [reply README.md:46]: I will add this in

File: README.md (hunk 1 above)
Comment (matt-crowson-moj) [reply README.md:41]: it happens here automagically here in the compose file -   `- $PWD/localstack:/docker-entrypoint-initaws.d`

**Review by matt-crowson-moj — APPROVED** [review]
LGTM

**Pull request merged**